seqr/views/apis/summary_data_api.py

```diff
@@ -274,7 +274,7 @@ def _search_new_saved_variants(family_variant_ids: list[FamilyVariantKey], user:
 def _get_metadata_projects(request, project_guid):
     is_analyst = user_is_analyst(request.user)
     is_all_projects = project_guid == ALL_PROJECTS
-    include_airtable = 'true' in request.GET.get('includeAirtable', '') and is_analyst and not is_all_projects
+    include_airtable = 'true' in request.GET.get('includeAirtable', '') and AirtableSession.is_airtable_enabled() and is_analyst and not is_all_projects
```

Comment:
the checkbox to add this param is already hidden for local installs so this would never actually be in the URL without airtable, but its still better to have explicit server side filtering